common/src/test/java/com/amazonaws/services/schemaregistry/common/configs/GlueSchemaRegistryConfigurationTest.java

```diff
@@ -97,6 +97,7 @@ public void testBuildConfig_fromMap_succeeds() {
     public void testBuildConfig_noRegionConfigsSupplied_throwsException() {
         Map<String, Object> configWithoutRegion = new HashMap<>();
         configWithoutRegion.put(AWSSchemaRegistryConstants.AWS_ENDPOINT, "https://test/");
+        System.setProperty("aws.profile", "");
```

Comment:
Is this needed for localstack to run?

Reply:
no, this is needed because we updated AWS SDK versions, so we need to unset the system property for the test to pass.